src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/Tasks/IncrementalCompileAnalyzer.cs

```diff
@@ -389,13 +390,22 @@ private void UpdateFileListForIncrementalBuild(List<FileUnit> modifiedXamlFiles)
                 {
                     numLocalTypeXamls = CompilerLocalReference.LocalMarkupPages.Length;
```

Comment:
Surely this variable was meant to be used in the for-loop two lines down.

Reply:
Yeah probably, no real reason for it that I can see, I'll just remove it.

Reply:
Don't remove it.  Use it.